test-harness/specs/validate-body-params/no-consumes.defined.oas2.txt

```diff
@@ -1,37 +0,0 @@
-====test====
-When I send a request to an operation that does not have a consumes defined
```

Comment:
@XVincentX, does the change in http-spec impact this test?

Reply:
Yeah this is something we need to discuss with @philsturgeon 

basically a spec file that has neither a global or a local consumes is wrong — and this test was testing a behaviour which should not happen.

If you do not define what your API can consume, we can't even try to parse and validate your body.

Reply:
This seems like reasonable thinking, but if we want to be friendly we can say “they might have meant JSON” perhaps?

Reply:
Given this change will affect other parts of Stoplight I've put a comment in the relevant PR, let's continue there @philsturgeon https://github.com/stoplightio/http-spec/pull/25#issuecomment-535035897